examples/source/advertising-analytics/Tracking_Core_Web_Vitals.html

```diff
@@ -99,3 +99,51 @@
       }
     </script>
   </amp-analytics>
+
+
+  <!--
+  ## Comparing performance on AMP Cache vs Origin
+  -->
+  <!--
+AMP Caches work to make sure your users have as great an expierence possible, but it is important to make sure that the Origin version of the page is just as performant. Your Core Web Vitals may be negativly impacted if your AMP page performs really great when served from an AMP Cache, but less well on the origin. A good way to make sure you are delivering a great experience for all your users is to keep track of how your pages' Core Web Vitals are performing on and off of the AMP Cache.
+
+We can build on our Google Analytics to do this. One of substitutions provided by amp-analytics is `ampdocHost` - this lets us track the URL of the page the user visited. We exand our earlier config using `extraUrlParams`. This lets us define a custom key and value for our analytics events. We need to [set up](https://support.google.com/analytics/answer/2709829) a new Custom Dimension, that will be used as the key. 
+
+![Screenshot of Google Analytics, showing the Custom Dimensions edit page](https://user-images.githubusercontent.com/465414/112056606-9638e680-8b2e-11eb-8d21-9f40faedc5f9.jpg)
+
+
+Once you have created the Custom Dimension to track the AMP Host, make sure you note the `Index` listed on the analytics page. This number is used in our configuration objects to connect `ampdocHost` to the Dimension you just created. Below  we assume your index was "1", (i.e. `cd1`). If your `Index` is 2, then use `cd2`, etc.
+  -->
+  <div>
+    <amp-analytics>
+      <script type="application/json">
+      {
+        "vars": {
+           "gtag_id": "UA-XXXXXXXXX-X",
+           "config": {
+             "UA-XXXXXXXXX-X": {
+                "groups": "default"
+             }
+           }
+        },
+        "extraUrlParams": {
```

Comment:
Seeing this example I ask myself why the CWV params are below triggers and the custom dimension at the root level.